assets/javascript/javascript.js

```diff
@@ -20,15 +22,15 @@ var losses = 0;
 var guessNumber = 15;
 
 
-function startGame() {
-
-    uessNumber = 15;
+function pickRandomWordFromWordBank() {
+    return wordBank[Math.floor(Math.random() * wordBank.length)];
+}
 
+function startGame() {
     // computer picks a random word from that array
-   
-    wordIndex = wordBank[Math.floor(Math.random() * wordBank.length)];
+    wordIndex = pickRandomWordFromWordBank()
```

Comment:
Extracting `wordBank[Math.floor(Math.random() * wordBank.length)];` into a function called  `pickRandomWordFromWordBank` makes this logic more readable, and understandable at first glance